src/main/java/uk/gov/hmcts/ccd/data/definition/HttpUIDefinitionGateway.java

```diff
@@ -77,16 +77,16 @@ public SearchResult getSearchResult(int version, String caseTypeId) {
     }
 
     @Override
-    public SearchInputDefinition getSearchInputDefinitions(int version, String caseTypeId) {
+    public SearchInputFieldsDefinition getSearchInputDefinitions(int version, String caseTypeId) {
```

Comment:
method rename please

Reply:
renamed to getSearchInputFieldDefinitions